src/components/main/main.js

```diff
@@ -110,6 +111,10 @@ app.component('main', {
             delete res.publicKey;
           }
 
+          this.notify.about(
```

Comment:
I use subtraction because Notification display how many `LSK` received, while `res.balance` and `this.account.get().balance` display total balance amount.